src/main.js

```diff
@@ -197,6 +207,8 @@ function appOnUp(service) {
 
         if (state === "0" || state === "1") {
             newElement.state = "unconfigured";
+        } else if (state === "2") {
+            newElement.state = "configuring";

```

Comment:
It's OK for now, but can be converted to `SystemStateEnum` + enum mapping object.

Reply:
Good point! 
I'll update this in the next PR, I have some planned work.